lesson_13/1_csv/remove_duplicates_from_csv.py

```diff
@@ -0,0 +1,15 @@
+#remove duplicates from first file
+with open('rmc.csv', 'r') as csvfile_1:
```

Comment:
It is always better to store such code in the separate methods. But your solution is working and okay)

Reply:
thank you, will take it to the next HW :)